src/test/java/api/loans/DeclareLostAPITests.java

```diff
@@ -108,6 +109,27 @@ void canDeclareItemLostWithComment() {
     assertThat(actualLoan, hasLoanProperty("declaredLostDate", isEquivalentTo(dateTime)));
   }
 
+  @Test
+  void declareItemLostFailsWhenEventPublishingFailsWithBadRequestError() {
+    UUID servicePointId = servicePointsFixture.cd1().getId();
+    final IndividualResource checkOut = checkOutFixture
+      .checkOutByBarcode(itemsFixture.basedUponNod(), usersFixture.jessica());
+
+    String comment = "testing";
+    ZonedDateTime dateTime = ClockUtil.getZonedDateTime();
```

Comment:
These variables are not needed, move the values to the builder below

Reply:
fixed